docs/get-started/roles-and-permissions/access_control.md

```diff
@@ -0,0 +1,8 @@
+# Introduction to Access Control in PMM
+
+!!! caution alert alert-warning "Caution"
+    PMM Access Control is currently in [technical preview](../details/glossary.md#technical-preview) and is subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+
+Access control in PMM allows you to manage who has access to individual Prometheus (Victoria Metics)  metrics based on **labels**. Thus, access management provides a standardized way of granting, changing, and revoking access to metrics based on the role assigned to the users.
```

Comment:
Is there any specific reason we mention lower level components here, such as Prometheus or VictoriaMetrics?

Reply:
@BupycHuk had asked me to mention it.

Reply:
yes, because we have 2 type of metrics: QAN and VM, at this moment we support Access Control only for VM metrics